sass/components/_links.scss

```diff
@@ -13,6 +13,21 @@ a {
     }
 }
 
+button.style--link {
```

Comment:
I thought it could be useful to have a global style for buttons that should look like links.